### PR TITLE
Feat: dashboard users chart frontend

### PR DESCRIPTION
**Pull request by Tymek**
Users chart with tooltip and legend

image

### CODE REVIEW

File: frontend/src/component/executiveDashboard/UsersChart/UsersChartComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,408 @@
...
+
+const UsersChartComponent: VFC = () => {
+    const theme = useTheme();
+    const { locationSettings } = useLocationSettings();
+    const data = createData(theme);
+    const options = createOptions(theme, locationSettings);
+
+    return (
+        <Paper sx={(theme) => ({ padding: theme.spacing(4) })}>
+            {/* <div>UsersChart</div> */}
```
Comment (kwasniew) [inline frontend/src/component/executiveDashboard/UsersChart/UsersChartComponent.tsx:391]: can be removed

Comment (Tymek) [inline frontend/src/component/executiveDashboard/UsersChart/UsersChartComponent.tsx:391]:
```suggestion
```

File: frontend/src/component/executiveDashboard/ExecutiveDashboard.tsx [hunk 2]
```diff
@@ -1,5 +1,33 @@
...
+    return (
+        <>
+            <Box sx={(theme) => ({ paddingBottom: theme.spacing(4) })}>
+                <PageHeader
+                    titleElement={
+                        <Typography variant='h1' component='h2'>
+                            Dashboard
+                        </Typography>
+                    }
+                    // subtitle='Succesfully synchronized: 01 Sep 2023 - 07:05:07'
```
Comment (kwasniew) [inline frontend/src/component/executiveDashboard/ExecutiveDashboard.tsx:23]: do we want to keep this comment?

File: frontend/src/component/executiveDashboard/UsersChart/UsersChartComponent.tsx [hunk 3]
```diff
@@ -0,0 +1,407 @@
...
+                position: 'bottom',
+            },
+            tooltip: {
+                callbacks: {
+                    title: (tooltipItems: any) => {
+                        const item = tooltipItems?.[0];
+                        const date =
+                            item?.chart?.data?.labels?.[item.dataIndex];
+                        return date
+                            ? formatDateYMD(date, locationSettings.locale)
```
Comment (kwasniew) [inline frontend/src/component/executiveDashboard/UsersChart/UsersChartComponent.tsx:346]: I'd pass timezone 'UTC' to this method. I exposed this option for the metrics. As a rule of thumb when we display daily results we do it in UTC. Hourly data is in local time.

File: frontend/src/component/executiveDashboard/UsersChart/UsersChartComponent.tsx [hunk 4]
```diff
@@ -0,0 +1,407 @@
...
+const createOptions = (theme: Theme, locationSettings: ILocationSettings) =>
+    ({
+        responsive: true,
+        plugins: {
+            legend: {
+                position: 'bottom',
+            },
+            tooltip: {
+                callbacks: {
+                    title: (tooltipItems: any) => {
```
Comment (kwasniew) [inline frontend/src/component/executiveDashboard/UsersChart/UsersChartComponent.tsx:341]: any type

Comment (kwasniew) [reply frontend/src/component/executiveDashboard/UsersChart/UsersChartComponent.tsx:341]: I'd either use default type TooltipItem as other places or pass unknown to a parser that produces a correct type

**Review by kwasniew — APPROVED** [review]
Nice! Some minor comments inside

**Comment by Tymek** [comment]
I'll fix comments in next PRs

**Pull request merged**